### ISSUE TITLE
Translation of our Contribution Guidelines

### ISSUE THREAD

**Issue by adamrusted** [simple-icons/simple-icons#10094]
### Kind of Issue

Improvement

### Description

I've seen before in other projects where their contribution guidelines are translated into the same languages as their website translations. This would make contributing a _lot_ easier for native speakers of those languages. I'm very aware when putting together the contribution guidelines there may be some colloquialisms sneaking into my writing!

Point being, I think we should have a separate `/CONTRIBUTING/[lang].md` file for each of our translations listed on the website - and ask the contributors who worked on the original translations for support, if they have the availability!

## Languages
- [ ] Spanish @mondeja
- [ ] French @mondeja 
- [ ] Italian @mondeja 
- [ ] Japanese @adamrusted / @ms0503
- [ ] Korean @sh-cho 
- [ ] Portuguese @mondeja 
- [ ] Ukrainian @Andygol
- [ ] Simplified Chinese @LitoMore 
- [ ] Traditional Chinese (Hong Kong) @LitoMore 
- [ ] Traditional Chinese (Taiwan) @LitoMore

**Comment by sh-cho** [comment]
Ok, will do

**Comment by adamrusted** [comment]
Legend, thanks @sh-cho!
If you are the first to contribute (I suspect you will be!) then if you just create a `CONTRIBUTING` folder, with `ko-KR.md`/`ko.md` as a file, and create a PR. I'll worry about linking up all translations in one master PR later down the line 😄

**Comment by sh-cho** [comment]
- https://github.com/sh-cho/simple-icons/tree/contributing-guide-kr

I'm working on it. I'll have it done in a few days 😃

**Comment by PeterShaggyNoble** [comment]
I am _very_ much in support of this. But I would suggest we hold off until #5628 is resolved, as you & I were discussing the other day, @adamrusted, so we don't end up asking our contributors to double up on the work. Combining this with what we were already thinking about, each language would then have its own subdirectory within the "contributing" directory with a README providing a brief overview with an index of the individual documents, with the README in the "contributing" directory itself providing an even briefer overview with an index of available languages. CONTRIBUTING.md, then, could just be a dupe of _that_ README.

**Comment by adamrusted** [comment]
@sh-cho - see above from Peter. I'd neglected the fact that we're looking to improve our guidelines as a whole. If you're able to keep the branch present for now, but we'll hold fire on merging until we're up to speed with the other issues/PRs

**Comment by mondeja** [comment]
I'm not very fan of translating static content as will get outdated soon without noticing it are updates at the source file. That's why, for Markdown content, I created a set of CLIs under the project mdpo but unfortunately it does not provide an API for Javascript.

And sorry, I don't have enough knowledge about translating Portuguese, French, nor Italian. If the translations are stable I will maintain the Spanish version.

**Comment by adamrusted** [comment]
I get where your hesitation comes from @mondeja, though it's somewhat restrictive of the project to only offer docs in English, if we've got all these languages available on our website. Perhaps it's best to slim down the list above to only languages maintainers are fluent in? That way we can call upon a maintainer to update docs whenever a change is made, without relying on outside contributors?

**Comment by LitoMore** [comment]
> I'm not very fan of translating static content as will get outdated soon without noticing.

Me too.

In the meantime, I don't think Chinese translations are necessary here. Our contributors should have a minimum level of English if they want to contribute code to us. The minimum level is to learn how to use and read Google Translate.

There is a very tragic case here https://github.com/standard/standard/tree/master/docs. The content of Standard's translation in each language version is different. I believe that their users will eventually return to English documentation.

**Comment by adamrusted** [comment]
What's the vibe then, we close this down and accept that we're only having docs in English?

**Comment by service-paradis** [comment]
> Our contributors should have a minimum level of English if they want to contribute code to us.

I agree with you on this. Without this minimum requirement, it means people opening PRs will expect us to interact with them in their language. It can complicate review processes.

**Comment by PeterShaggyNoble** [comment]
> Our contributors should have a minimum level of English

This is something, in general on the web, that I am strongly opposed to: English being the de facto default language.

> The minimum level is to learn how to use and read Google Translate

Machine translations, though, by & large, absolutely suck!

> There is a very tragic case here https://github.com/standard/standard/tree/master/docs. The content of Standard's translation in each language version is different.

This, though, is a very valid concern. Unless it's in a language someone on the team speaks, which is a good few, we can't verify the accuracy of  a translation nor that it hasn't simply been run through a machine translator. And the only way I can see around that is to require a minimum of 2 people working on each translation, which would take a bit of work to coordinate. So, the question becomes: Which is the lesser of 2 evils - _not_ proving any translations, or providing inaccurate translations? 🤔
